### ISSUE TITLE
Acessibility fixes

### ISSUE THREAD

**Issue by pdpinch** [mitodl/mitxonline#150]
#### Acceptance Criteria:

- [x] Fix duplicate H1 tags on home page (there can be only one H1 per page)
- [x] Use empty ALT tags for course cards on home page by setting `alt=""`. The thumbnails are purely decorative and no alt tag would be preferable to the repetitive ones we have now. 
- [x] sign in form input tag is missing an id attribute, `id="email"` which is necessary to for the `<label>` tag to function. 
  - [x] review any other forms for missing `id` attributes and/or missing `<label>` tags. 
- [x] When an error message is updated in forms, the container needs a `role="alert"` attribute
- [x] input tags should have an `aria-desribed-by` attribute, set to the id of the container of any error messages. 
- [x] Change `aria-label` attributes on "Sign In" and "Create Account" buttons to match the text of the buttons, instead of them both being "Login". 
- [x] Change the (duplicate) alt tag on the MIT logo from "MITx Online" to "MIT Home page" 
- [x] #171 Use the Lighthouse tool to review accessibility of all other page templates and make any necessary changes

**Comment by arslanashraf7** [comment]
@pdpinch Looks like we are running 100% accessibility as per the comment on the last part of this epic https://github.com/mitodl/mitxonline/issues/171#issuecomment-926438335. 

I think we can close this ticket, We can reopen this if needed.
